qa/L0_response_cache/test.sh

```diff
@@ -31,11 +31,56 @@ TEST_LOG="./response_cache_test.log"
 UNIT_TEST="./response_cache_test --gtest_output=xml:response_cache.report.xml"
 export CUDA_VISIBLE_DEVICES=0
 
+REPO_VERSION=${NVIDIA_TRITON_SERVER_VERSION}
+if [ "$#" -ge 1 ]; then
+    REPO_VERSION=$1
+fi
+if [ -z "${REPO_VERSION}" ]; then
+    echo -e "Repository version must be specified"
+    echo -e "\n***\n*** Test Failed\n***"
+    exit 1
+fi
+if [ ! -z "$TEST_REPO_ARCH" ]; then
+    REPO_VERSION=${REPO_VERSION}_${TEST_REPO_ARCH}
+fi
 # Only localhost supported in this test for now, but in future could make
 # use of a persistent remote redis server, or similarly use --replicaof arg.
 export TRITON_REDIS_HOST="localhost"
 export TRITON_REDIS_PORT="6379"
 REDIS_LOG="./redis-server.unit_tests.log"
+ENSEMBLE_CACHE_TEST_PY="./ensemble_cache_test.py"
+SERVER=/opt/tritonserver/bin/tritonserver
+CLIENT_LOG="./client.log"
+TEST_RESULT_FILE='test_results.txt'
+SERVER_LOG=./inference_server.log
+RESET_CONFIG_FUNCTION="_reset_config_files"
+CACHE_SIZE=10840
```

Comment:
Add comment on size choice - why this number? what's the requirements/goal?

Reply:
I used a random number. Something that worked for me. 
For test_cache_insertion testcase, I used CACHE_SIZE=200 because the data to be inserted is more than 200. Found the value through calculations and testing.